community/event_management/doctype/talk/talk.json

```diff
@@ -61,54 +56,29 @@
    "label": "Attachment"
   },
   {
-   "fieldname": "speaker",
-   "fieldtype": "Link",
-   "label": "Speaker",
-   "options": "Speaker"
-  },
-  {
-   "default": "Pending",
+   "default": "Applied",
    "fieldname": "status",
    "fieldtype": "Select",
    "in_list_view": 1,
    "label": "Status",
-   "options": "Pending\nApproved\nRejected"
-  },
-  {
-   "fieldname": "column_break_5",
-   "fieldtype": "Column Break"
-  },
-  {
-   "fieldname": "section_break_9",
-   "fieldtype": "Section Break"
+   "options": "Applied\nPending\nApproved\nRejected\nPrevious Talk"
```

Comment:
What's the difference between Applied and Pending? Why do we need a status called Previous Talk?